smalllog.txt

```diff
@@ -0,0 +1,2238 @@
+D 21:33:52.1938781 GameState.DebugPrintPowerList() - Count=103
+D 21:33:52.1973520 GameState.DebugPrintPower() - CREATE_GAME
+D 21:33:52.1973520 GameState.DebugPrintPower() -     GameEntity EntityID=1
+D 21:33:52.1998288 GameState.DebugPrintPower() -         tag=CARDTYPE value=GAME
+D 21:33:52.2003246 GameState.DebugPrintPower() -         tag=ZONE value=PLAY
+D 21:33:52.2003246 GameState.DebugPrintPower() -         tag=ENTITY_ID value=1
+D 21:33:52.2003246 GameState.DebugPrintPower() -         tag=937 value=2
+D 21:33:52.2003246 GameState.DebugPrintPower() -     Player EntityID=2 PlayerID=1 GameAccountId=[hi=144115198130930503 lo=14514236]
```

Comment:
If this is your actual account ID, it might be a good idea to remove it. I'm not sure what somebody could do with it, but better safe than sorry. 🙂